centinel/views.py

```diff
@@ -171,17 +171,76 @@ def get_user_specific_content(folder, filename=None, json_var=None):
     if not client.has_given_consent:
         flask.abort(418)
 
+    # all of the scheduler files are combined together here.
+    # this is run every time the experiment list or "scheduler.info"
+    # is requested.
+    if (json_var == "experiments" and 
+        (filename is None or filename == "scheduler.info")):
+        client_scheduler_filename = os.path.join(config.experiments_dir,
+                                                 username, "scheduler.info")
+        country_scheduler_filename = os.path.join(config.experiments_dir,
+                                                  client.country, "scheduler.info")
+        global_scheduler_filename = os.path.join(config.experiments_dir,
+                                                 "global", "scheduler.info")
+
+        freqs = {}
+        if os.path.exists(client_scheduler_filename):
+            with open(client_scheduler_filename, 'r') as file_p:
+                freqs.update(json.load(file_p))
+        if os.path.exists(country_scheduler_filename):
+            with open(country_scheduler_filename, 'r') as file_p:
+                freqs.update(json.load(file_p))
+        if os.path.exists(global_scheduler_filename):
```

Comment:
I would suggest that we flip the order here- we want the client scheduler stuff to have precedence over the global file. That way, we can override the global scheduler with local settings if we want. If you reverse the order here, it will fix the problem.